src/vsr/replica.zig

```diff
@@ -4546,7 +4706,14 @@ pub fn ReplicaType(
             assert(message.header.commit == self.commit_min);
             DVCQuorum.verify_message(message);
 
-            self.send_message_to_replica(self.primary_index(self.view), message);
+            self.send_message_to_other_replicas(message);
+
+            if (self.replica == self.primary_index(self.view) and
+                self.do_view_change_from_all_replicas[self.replica] == null)
+            {
+                self.send_message_to_replica(self.replica, message);
+                defer self.flush_loopback_queue();
```

Comment:
The `defer` here doesn't do anything, it is more of a hint that `flush_loopback_queue` shouldn't have anything after it. (We do the same elsewhere for `flush_loopback_queue`).